packages/core/test/create-request-client.js

```diff
@@ -284,18 +299,10 @@ describe('request client', () => {
       testLogger
     );
     const request = createAppRequestClient(inputWithAfterMiddleware);
-    request({ url: 'https://httpbin.org/get' })
-      .then(responseBefore => {
-        const response = JSON.parse(JSON.stringify(responseBefore));
+    const response = await request({ url: 'https://httpbin.org/status/400' });
```

Comment:
Updated test for not using the default if you define a custom `afterResponse` (with no `throwForStatus`)